README.md

```diff
@@ -53,4 +53,41 @@ An overview of this method is discussed in the [full documentation](https://mncr
 This code contains a Julia implementation of the MATLAB code included as supplementary material with [Crowe & Johnson 2024](https://doi.org/10.1017/jfm.2024.619) and also includes a version of the previously unrealeased SQG version discussed in [Johnson & Crowe 2023](https://doi.org/10.1017/jfm.2023.87) and [Crowe & Johnson 2023](https://doi.org/10.1017/jfm.2023.607).
 For those interested in the original (layered QG only) implementation, it can be found [here](https://github.com/mncrowe/QGDipoles.m). 
 
- 
+## Contributing
+
+Issue should be submitted [here](https://github.com/mncrowe/QGDipoles.jl/issues/) and discussions or questions [here](https://github.com/mncrowe/QGDipoles.jl/discussions).
+
+We follow the [ColPrac guide for collaborative practices](https://github.com/SciML/ColPrac). New contributors should make sure to read that guide. Below are some additional practices we follow.
+
+### Editing
+
+The source files are stored in `src/`. It is recommended to open Julia with the `QGDipoles` project active by running `julia --project=.` from the root directory. Be careful not to add unnecessary packages to `Project.toml`.
+
+### Formatting
+
+This codebase is formatted using `JuliaFormatter.jl`. Formatting can be done by entering the following into the Julia REPL from the root directory:
+```julia
+import JuliaFormatter
+format(".")
+```
+`JuliaFormatter.jl` should not be added to the package dependencies of `QGDipoles.jl`.
+
+### Documentation
+
+The documentation is stored within `docs/` and can be built using `Documenter.jl` by running `docs/make.jl`. Full instructions for using `Documenter.jl` are available [here](https://documenter.juliadocs.org/stable/man/guide/). The documentation can be re-built by opening Julia using `julia --project=docs/` then entering the following into the REPL:
+```julia
+Pkg.develop(".")
```

Comment:
```suggestion
Pkg.develop(".")  # this adds the local copy of `QGDipoles.jl` to the `docs` environment 
```

You could also mention that users shouldn't push the `Project.toml` file containing `QGDipoles.jl` dependency.